internal/secrets/gcp_secret_manager.go

```diff
@@ -19,15 +19,30 @@ package secrets
 import (
 	"context"
 	"fmt"
+	"sync"
+	"time"
 
 	"github.com/google/exposure-notifications-server/internal/logging"
 
 	secretmanager "cloud.google.com/go/secretmanager/apiv1"
 	secretmanagerpb "google.golang.org/genproto/googleapis/cloud/secretmanager/v1"
 )
 
+var (
+	// cacheDuration is how long to cache a secret.
+	cacheDuration = 5 * time.Minute
```

Comment:
should probably be a secret manager config so that this can be tuned - can be a follow up / file an issue

Reply:
Done